pkgs/top-level/python-packages.nix

```diff
@@ -303,6 +303,8 @@ in {
   cleo = callPackage ../development/python-modules/cleo { };
 
   clikit = callPackage ../development/python-modules/clikit { };
+  
+  cfn_flip = callPackage ../development/python-modules/cfn_flip { };
```

Comment:
f comes before l if I remember correctly

Reply:
yeah right hhhhhhh @dotlambda

Reply:
still wrong, as is the commit message
and I forgot to tell you to normalize attribute and filename to `cfn-flip`.

Reply:
yeah seems like i didn't `git add .`